tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Tests for Block Bindings API "core/post-meta" source.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.5.0
+ *
+ * @group blocks
+ * @group block-bindings
+ */
+class Tests_Block_Bindings_Post_Meta_Source extends WP_UnitTestCase {
+
+	/**
+	 * Sets up each test method.
+	 */
+	public function set_up() {
+		global $post;
+
+		parent::set_up();
+
+		$post = self::factory()->post->create_and_get( array() );
+		setup_postdata( $post );
+	}
+
+	/**
+	 * Tear down each test method.
+	 */
+	public function tear_down() {
+		// Removes custom fields registered by test cases.
+		$meta_keys = get_registered_meta_keys( 'post', '' );
+		foreach ( $meta_keys as $meta_key_name => $meta_key_value ) {
+			unregister_meta_key( 'post', $meta_key_name );
```

Comment:
Is it safe to unregister all meta keys for the `post` type?

Reply:
I added a check to unregister only the fields with "tests" in the name: [link](https://github.com/WordPress/wordpress-develop/pull/6197/commits/1fb2c47758c6deff8342b85b20d7ccaf15dd865c).